backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
```

Comment:
where did you get this piece of knowledge? I was under impression that the structure is more or less plain (two level of hierarchy project and subfolder). I you want just to make as general as possible, I wouldn't do so because it complicates schema and I don't see benefits for MVP. @gromakovsky what do you think?

In general recursive schema are used in case of tree hierarchy as you said, like folder structure but we don't seem to need this

Reply:
@pva701 what's the complication to track parent ID?

I can imagine that a hypothesis will have several attempts to check it as several sub-folders of sub-folder.

Reply:
(How) Do we track subfolders?

Reply:
We don't, but we should keep in mind that they want hierarchy.

Reply:
Okay, there was an update saying that we don't need a hierarchy for now, so we can resolve this conversation and get back to it some time later.

Reply:
@pva701 I propose to add a comment saying that we'll need a hierarchy some time later, so that we keep it in mind.